tasks/reproducibility-xfel.tex

```diff
@@ -3,42 +3,165 @@
+\emph{Summary:}
+
+  European XFEL is a research facility that provides X-ray Free
+  Electron Laser (XFEL) light to image structures at the nanoscale. It
+  is currently the world's most brilliant laser, created in a 3.4km
+  long tunnel, and supporting user experiments since September
+  2017. These imaging capabilities from European XFEL and similar
+  services from synchrotron and neutron sources, underpin lots of
+  fundamental and applied research, in domains ranging from fundamental
+  physics and material science to biochemistry and drug design.
+
+  All of the data recorded at European XFEL will be made freely
+  available after an embargo period of three years
+  \cite{EuXFEL-datapolicy-2017}. This provides scientific transparency
+  and is expected to enable better exploitation of the data, as more
+  researchers than those conducting the experiments have access to the
+  results. If the analysis steps are not carefully recorded, there is a risk
+  that the necessary understanding of the data is lost by the time it
+  is made public or subsequently, greatly reducing its scientific
+  value.
+
+  We are keen to complement this open data access to the actual data
+  with open access to reproducible data analysis, to confirm
+  conclusions drawn and to significantly lower the barriers for
+  re-analysis with new tools or for new research purposes.
+
+  A task in the EC funded project Photon and Neutron Open Science
+  Cloud (PaNOSC) is using the Jupyter Ecosystem tools as they are in
+  2019 to provide interactive data analysis services to complement the
+  data: through use of Jupyter Notebook and exploitation of the
+  mybinder.org service, this activity will reduce the barrier for
+  interactively exploring the data, understanding and making use of
+  the data, and to do this through a central portal such as EOSC.
+
+  Here, we want to combine and use the new developments of this
+  proposal to enable new qualities of open science services, and to
+  demonstrate the potential impact of these improvements for EOSC.
+
+  \medskip \emph{Context}: The very first experiments at European XFEL
+  produced as little as 45 TeraByte of data on average, but as the
+  facility develops, the amount of data produced per time is expected
+  to grow substantially: Given the rate of light pulses, there is the
+  potential to produce up to a PetaByte of data within the beam time
+  of one experiment. These significant amounts of data need to be
+  complemented by complicated workflows to convert the data into
+  insight through data analysis. Derived results of such data analysis
+  are typically much smaller in size and useful to archive together
+  with the raw data. To explain how they have been obtained, the
+  particular workflow of data analysis also needs to be archived.
+
+  \medskip \emph{Vision}: At European XFEL we will use Jupyter Notebooks to facilitate
+  this workflow: the simplest model would be to use one Notebook per
```

Comment:
'one notebook', lowercase n. The software is the Jupyter Notebook (a proper name), the documents are notebooks (a regular noun).